tutorials/integrations/main.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import importlib
+import logging
+from integrations.common.helpers import write_results, read_models_list, parse_results
+from integrations.common.helpers import find_modules
+from integrations.common.consts import MODEL_NAME, MODEL_LIBRARY, OUTPUT_RESULTS_FILE, TARGET_PLATFORM_NAME, \
+    TARGET_PLATFORM_VERSION
+
+
+def argument_handler():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--model_name', '-m', type=str, required=False,
+                        help='The name of the pre-trained model to run')
+    parser.add_argument('--model_library', type=str, required=False,
+                        help='The library that contains the pre-trained model')
+    parser.add_argument('--validation_dataset_folder', type=str, required=False,
+                        help='Path to the validation dataset')
+    parser.add_argument('--representative_dataset_folder', type=str, required=False,
+                        help='Path to the representative dataset used for quantization')
+    parser.add_argument('--num_representative_images', type=int, default=1024,
+                        help='Number of images for representative dataset')
+    parser.add_argument('--dataset_name', type=str, default='',
+                        help='Datset name for comment')
+    parser.add_argument('--target_platform_name', type=str, default='default',
+                        help='Specifies a target platform capabilites (tpc) name')
+    parser.add_argument('--target_platform_version', type=str, default=None,
+                        help='Specifies a target platform capabilites (tpc) version')
+    parser.add_argument('--batch_size', type=int, default=32,
+                        help='Batch size for accuracy evaluation')
+    parser.add_argument('--random_seed', type=int, default=0,
+                        help='Random seed')
```

Comment:
Improve help message (also, are you using the random seed argument to define the seed somewhere? I couldn't find it)